distribution/docs/src/main/resources/_contents/_solr-contents/managing-solr-contents.adoc

```diff
@@ -309,3 +309,50 @@ Here are some helpful suggestions for setting up data backups and recoveries:
 * Run a scheduler program that calls the backup URL on a timed basis.
 * Put indicators in place that can detect when data corruption may have occurred.
 * Testing a backup before recovery is possible. A replicated "staging" Solr server instance can be stood up, and the backup can be copied to that system for testing before moving it to the "production" system.
+
+=== High Availability (Optional)
+
+==== SolrCloud
+
+SolrCloud provides distributed capabilities of a highly available, fault tolerant cluster of Solr servers.
+
+===== Prerequisites
+
+Puppet installed and configured.
+
```

Comment:
Should we mention puppet at all at this point? I thought we weren't using puppet quite yet. But maybe I'm wrong.